improver/feels_like_temperature.py

```diff
@@ -129,8 +129,8 @@ def _calculate_apparent_temperature(
     This function looks up a value for the saturation vapour pressure of
     water vapour using the temperature and a table of values. These tabulated
     values are found using lookup_svp and are corrected to the saturated
-    vapour pressure in air using pressure_correct_svp, both functions are from
-    the WetBulbTemperature plugin which makes use of the Goff-Gratch method.
+    vapour pressure in air using pressure_correct_svp, taken from the Goff-Gratch
+    method.
```

Comment:
As far as I can see there are no methods called `lookup_svp` or `pressure_correct_svp` anymore, so I think this needs a slightly larger rewrite.

Reply:
I missed that, thanks. Updated to the new method names.